lib/bambora/client/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Bambora
   class Client
-    VERSION = '0.6.0'
+    VERSION = '0.7.0'
```

Comment:
Version bump. Last time I forgot to create a `tag` and release to `ruby-gems` but this time I have the power! ⚔️